Automattic/wp-calypso
# Export: Fetch available settings for customizing export content

This is a straightforward Redux fetch-receive-fail pattern for fetching the available advanced settings for export. The majority of the 300+ additions are the associated automated tests.

The fetched data will (in a future PR) be used to populate the following menus:

<img width="588" alt="wordpress_com" src="https://cloud.githubusercontent.com/assets/416133/12668334/27b11842-c6a1-11e5-91df-2adce3a56f38.png">

Actually populating the advanced settings menus in the UI is outside of the scope of this PR.

### How to test
Run the following command and ensure the automated tests pass:
`(cd client/state/site-settings/exporter && make test)`
